django/db/models/fields/__init__.py

```diff
@@ -718,6 +718,14 @@ def unique(self):
     def db_tablespace(self):
         return self._db_tablespace or settings.DEFAULT_INDEX_TABLESPACE
 
+    @property
```

Comment:
Any reason to make this a property instead of a class level attribute?

Reply:
The default implementation will be more complicated in the future when we will add db defaults. I think we can leave it is as a property.